StreamChatUITestsAppUITests/Robots/UserRobot.swift

```diff
@@ -212,6 +212,81 @@ extension UserRobot {
         MessageListPage.list.press(forDuration: 0.1, thenDragTo: topMessage)
         return self
     }
+    
+    @discardableResult
+    func openComposerCommands() -> Self {
+        if MessageListPage.ComposerCommands.cells.count == 0 {
+            MessageListPage.Composer.commandButton.wait().tap()
+        }
+        return self
+    }
+    
+    @discardableResult
+    func sendGiphy(useComposerCommand: Bool = false, shuffle: Bool = false) -> Self {
+        let giphyText = "Test"
+        if useComposerCommand {
+            openComposerCommands()
+            MessageListPage.ComposerCommands.giphyImage.wait().tap()
+            sendMessage("\(giphyText)")
+        } else {
+            sendMessage("/giphy\(giphyText)")
+        }
+        if shuffle { tapOnShuffleGiphyButton() }
+        tapOnSendGiphyButton()
+        return self
+    }
+    
+    @discardableResult
+    func replyWithGiphy(
+        useComposerCommand: Bool = false,
+        shuffle: Bool = false,
+        messageCellIndex: Int = 0
+    ) -> Self {
+        selectOptionFromContextMenu(option: .reply, forMessageAtIndex: messageCellIndex)
+        sendGiphy(useComposerCommand: useComposerCommand, shuffle: shuffle)
+        return self
```

Comment:
nit: since all methods already return `self` maybe we can leverage it? (the same applies for other helpers)
```suggestion
        return self
            .selectOptionFromContextMenu(option: .reply, forMessageAtIndex: messageCellIndex)
            .sendGiphy(useComposerCommand: useComposerCommand, shuffle: shuffle)
```

Reply:
I like it! Updated 🙂